src/modules/module_utils.c

```diff
@@ -413,6 +422,54 @@ void module_strip_punctuation_default(char *buf)
 	module_strip_punctuation_some(buf, "~#$%^&*+=|<>[]_");
 }
 
+gchar *module_multicases_string_replace(
+	const gchar *text,
```

Comment:
`text` is never freed. The caller expects `module_multicases_string_replace` to free it (and that should be documented so)